api-docs/openapi.json

```diff
@@ -87,115 +87,115 @@
     }
   ],
```

Comment:
I hate this json file 😭 looking forward to a day when we delete it #wishful-thinking